client/src/components/NeighborhoodMap.jsx

```diff
@@ -1,34 +1,47 @@
 import React from 'react';
 import Details from './Details';
 import { Map, GoogleApiWrapper } from 'google-maps-react';
-const googleMapApi = require('../../../googleMapConfig');
+import '../../dist/index.css';
+// DO NOT COMMIT ME TO VERSION CONTROL!!!!!
```

Comment:
We can remove this comment, and I trust you did appropriate research about including the API key in this file. Just please be careful.